packages/@react-spectrum/list/chromatic/ListView.chromatic.tsx

```diff
@@ -10,55 +10,124 @@
  * governing permissions and limitations under the License.
  */
 
-import {ActionButton} from '@react-spectrum/button';
+import {ActionGroup} from '@react-spectrum/actiongroup';
+import {ActionMenu} from '@react-spectrum/menu';
+import Add from '@spectrum-icons/workflow/Add';
 import {Content, View} from '@react-spectrum/view';
-import {Flex} from '@react-spectrum/layout';
-import {Heading} from '@react-spectrum/text';
+import Delete from '@spectrum-icons/workflow/Delete';
+import Folder from '@spectrum-icons/workflow/Folder';
+import {generatePowerset} from '@react-spectrum/story-utils';
+import {Grid, repeat} from '@react-spectrum/layout';
+import {Heading, Text} from '@react-spectrum/text';
 import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import Info from '@spectrum-icons/workflow/Info';
 import {Item, ListView} from '../';
+import {Link} from '@react-spectrum/link';
 import {Meta, Story} from '@storybook/react';
 import React from 'react';
 
-let flatOptions = [
-  {name: 'row 1'},
-  {name: 'row 2'},
-  {name: 'row 3'}
+// TODO: add overflow mode and thumbnails when those PRs are merged
+let states = [
+  {isQuiet: true},
+  // No visual difference between single and multiple
+  {selectionMode: 'single'},
+  {density: ['compact', 'spacious']},
+  {selectionStyle: 'highlight'}
 ];
 
-let withButtons = [
-  {name: 'row 1', button: 'Button 1'},
-  {name: 'row 2', button: 'Button 2'},
-  {name: 'row 3', button: 'Button 3'}
-];
+let combinations = generatePowerset(states);
+let chunkSize = Math.ceil(combinations.length / 3);
+
+function shortName(key, value) {
+  let returnVal = '';
+  switch (key) {
+    case 'isQuiet':
+      returnVal = 'quiet';
+      break;
+    case 'selectionMode':
+      returnVal = `sm: ${value === undefined ? 'none' : value}`;
+      break;
+    case 'density':
+      returnVal = `den: ${value === undefined ? 'regular' : value}`;
+      break;
+    case 'selectionStyle':
+      returnVal = 'highlight';
+      break;
+  }
+  return returnVal;
+}
 
 const meta: Meta = {
   title: 'ListView',
   component: ListView,
   parameters: {
+    chromaticProvider: {colorSchemes: ['light', 'dark'], locales: ['en-US'], scales: ['medium', 'large'], disableAnimations: true},
```

Comment:
did it just extend too far to do lightest and darkest?

Reply:
yeah, would mean more story splitting for performance. I'm beginning to think that perhaps we should have a separate story per combo, might be a lot though

Reply:
Looking at https://github.com/storybookjs/storybook/issues/9828, doesn't seem to be a great way to do this w/o going back to the storiesOf api. I'm leaning towards keeping this PR as is since we are looking to upgrade our storybook version and waiting for overflowMode to go in so we can get a relatively stable set of combinations per story (otherwise any changes to the power set will result in a difficult to comprehend diff).